train.py

```diff
@@ -112,6 +112,8 @@ def train(hyp, opt, device, callbacks):  # hyp is path/to/hyp.yaml or hyp dictio
         data_dict = data_dict or check_dataset(data)  # check if None
     train_path, val_path = data_dict['train'], data_dict['val']
     nc = 1 if single_cls else int(data_dict['nc'])  # number of classes
+    channels = data_dict['channels'] if 'channels' in data_dict else []  # the channels of data more than RGB
```

Comment:
What is difference between channels and ch arguments?
It seems ch is for number of channels; but, what about channels? and where is it used?